generic/tests/vlan.py

```diff
@@ -153,6 +170,7 @@ def flood_ping(src, dst):
     vms.append(env.get_vm("vm2"))
     for vm_ in vms:
         vm_.verify_alive()
+    bridge_vlan_config()
```

Comment:
Could you separate the old cases with your newly added case?
I mean, the old case is refer to RHEL7-95961 and do not need to set switch.
Moreover, in this case, no need to do file transfer test.
So my opinion is to separate them. You can add variants in the cfg.

Reply:
And are you going to support Windows guest in this PR?
Or add the support later?

Reply:
Thanks @hereischen @xiywang , will rewrite this case to bridge only vlan test script as Xiyue suggested, please help review it later again. Thanks so much!